code/clue.js

```diff
@@ -7,53 +7,332 @@ const mrGreen = {
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'knife',
 }
+console.log(mrGreen.description);
+
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is quick-witted, who lost his medical license for having an affair with one of his patients.',
+  age: 35,
+  image: 'assets/plum.png',
+  occupation: 'Professor',
+  favouriteWeapon: 'rope',
+}
+console.log(professorPlum.description);
```

Comment:
Before push, we should delete all console logs.
Needs to be fixed in whole file